sdks/python/apache_beam/io/external/xlang_kafkaio_it_test.py

```diff
@@ -28,64 +28,60 @@
 import time
 import typing
 import unittest
+import uuid
 
 import apache_beam as beam
 from apache_beam.io.kafka import ReadFromKafka
 from apache_beam.io.kafka import WriteToKafka
-from apache_beam.metrics import Metrics
 from apache_beam.testing.test_pipeline import TestPipeline
-
-
-class CrossLanguageKafkaIO(object):
```

Comment:
Probably keep this so that this pipeline can be reused. (for example, I believe we use this for Google internal testing).

Reply:
Oh, I didn't know about that. I meant to reduce the amount of code. I'll bring it back then.